lib/view/view_templates/html_text_card_view_template.dart

```diff
@@ -13,17 +13,26 @@ class HtmlTextCardViewTemplate extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        backgroundColor: bgColor ?? AppColors.backgroundBlue,
-        appBar: AppBar(
-            backgroundColor: AppColors.appBarBackground,
-            iconTheme: Styles.appBarIconTheme,
-            title: Text(title, style: Styles.appBarTextStyle)),
-        body: SingleChildScrollView(
-            child: Padding(
-                padding: const EdgeInsets.all(12),
-                child: Card(
-                    child: HtmlWidget(html,
-                        textStyle: Styles.cardDescriptionTextStyle,
-                        bodyPadding: const EdgeInsets.all(6))))));
+      backgroundColor: bgColor ?? AppColors.backgroundBlue,
+      appBar: AppBar(
```

Comment:
Seem to repeat this Appbar config several diff places, can we just have a builder function or custom widget for it t keep it more DRY ?

Reply:
Yep afraid no way around it right now.

Reply:
But we could have a single function that returns this appbar widget

Reply:
@maks Yes we could and should, but would prefer to do that in a different wider PR if thats OK. We have too many AppBar widgets anyway. We have a couple of key screen layouts that can use a new custom widget, that contains the AppBar. 

So thats a bigger chunk of work, so prefer to keep this to the scope of just OS fixups. This PR is getting a bit big as it is.